client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/Sections/Details.tsx

```diff
@@ -282,7 +223,7 @@ export const Details = ({
             plugins.requestRequisitionLine?.editViewField?.map(
               (Field, index) => <Field key={index} line={line} />
             )}
-          <RequestStats draft={draft} />
+          {draft && <RequestStats draft={draft} />}
```

Comment:
Chart is blank when adding new lines... I think because line doesn't exist on requisition yet 🫠 Let's have a quick brainstorm? I think might be best to send item id to backend, it can use that if requisition line id doesn't exist yet?

Also, per Roxy's comment we should have all the charts here, not just consumption history :)

https://github.com/user-attachments/assets/1efef230-0544-434c-ae08-f45a887bb665

Reply:
Charts shouldn't show if the line is a draft line